DAQ/src/CaloHitsFromDTCEvents_module.cc

```diff
@@ -320,16 +320,27 @@ void art::CaloHitsFromDataDTCEvents::analyze_calorimeter_(
       float time = thisHitPacket.Time + peakIndex * digiSampling_ + timeCalib_[SiPMID];
 
       bool isCaphri = offlineId.crystal().isCaphri();
+      if(!isCaphri) {
+        if(offlineId.crystal().disk() == 0) ++nhits_d0;
```

Comment:
if nhits were an array this line would be simply  ++nhits[offlineId.crystal().disk()]

Reply:
I've simplified this as suggested, and also in `CaloHitMakerFast`